### DISCUSSION TITLE
DEE6D204-2B1C-4589-A63B-42D277F97146

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#113] [General]
# DEE6D204-2B1C-4589-A63B-42D277F97146

Rao is the name of the red dwarf (supergiant in other various media) where Superman's homeworld, the planet Krypton, is located. In the 2012 comic, Superman#14 

https://celestia.mobi/addon?item=DEE6D204-2B1C-4589-A63B-42D277F97146

**Comment by EricNelson-sketch** [comment]
It's a pretty decent texture for the planet in the beloved Superman.
